.goreleaser.yaml

```diff
@@ -40,7 +40,7 @@ builds:
     goos:
       - 'darwin'
       - 'linux'
-      - 'windows'
+      # - 'windows'  # Coming some day
```

Comment:
Lol ref #70? Or add a comment to #70 to also update the release automation (just so we don't forget)

Reply:
Updated #70 👍